bin/commands/internal/start/prepare/index.ts

```diff
@@ -405,7 +410,7 @@ export function execute() {
   common.printFormattedInfo("ZWELS", "zwe-internal-start-prepare", `Zowe version: v${zoweVersion}`);
   common.printFormattedInfo("ZWELS", "zwe-internal-start-prepare", `build and hash: ${runtimeManifest.build.branch}#${runtimeManifest.build.number} (${runtimeManifest.build.commitHash})`);
 
-
+os.sleep(30000);
```

Comment:
magic number, also not clear why this sleep is necessary

Reply:
it was for testing, now removed.